Cloud Monitoring: MQL support (#26551)
grafana/grafana: docs/sources/datasources/cloudmonitoring.md

SLO queries use the same [alignment period functionality as metric queries]({{< relref "#metric-queries" >}}).

### MQL (Monitoring Query Language) queries

> **Note:** Only available in Grafana v7.4+.

> The MQL query builder in the Google Cloud Monitoring data source allows you to display MQL results in time series format. To get an understanding of the basic concepts in MQL, refer to [Google Cloud Monitoring documentation](https://cloud.google.com/monitoring/mql).

Reviewer: Please change that link text as follows so that it is WYSIWYG:

[Introduction to Monitoring Query Language](https://cloud.google.com/monitoring/mql)

Reviewer 2: fixed! thanks for feedback @osg-grafana

SLO queries use the same [alignment period functionality as metric queries]({{< relref "#metric-queries" >}}).

### MQL (Monitoring Query Language) queries

> > Only available in Grafana v7.x+

Reviewer 3: ```suggestion
> **Note:** Only available in Grafana v7.4+.
```

> The MQL query builder in the Google Cloud Monitoring data source allows you to display MQL result in time series format. To get an understanding of the basic concepts in MQL, please refer to Google Cloud Monitoring's [official docs](https://cloud.google.com/monitoring/mql).

Reviewer 3: ```suggestion
The MQL query builder in the Google Cloud Monitoring data source allows you to display MQL result in time series format. To get an understanding of the basic concepts in MQL, refer to [Google Cloud Monitoring documentation](https://cloud.google.com/monitoring/mql).
```

> #### How to create an MQL query

Reviewer 3: ```suggestion
#### Create an MQL query
```

To create an MQL query, follow these steps:

> 1. Choose the option **Monitoring Query Language (MQL)** in the **Query Type** dropdown.

Reviewer 3: ```suggestion
1. In the **Query Type** list, click **Monitoring Query Language (MQL)**.
```

Reviewer 3: When writing instructions, it is a good idea to tell people where to do the thing before you tell them what to do.

> 1. Choose a project from the **Project** dropdown.

Reviewer 3: ```suggestion
1. Choose a project from the **Project** list.
```

> 1. Specify [MQL](https://cloud.google.com/monitoring/mql/query-language) to query field.

Reviewer 4: The following wording seems more correct/understandable to me:
```suggestion
1. Specify [MQL](https://cloud.google.com/monitoring/mql/query-language) in the query field.
```

> 1. Specify [MQL](https://cloud.google.com/monitoring/mql/query-language) to query field.

Reviewer 3: Unclear. Am I selecting MQL from a list? Am I typing the entry somewhere? If so, where? 

Also, the link here is confusing. Why are you sending me to another website?

Author: User edit MQL in text area.
![image](https://user-images.githubusercontent.com/224552/97247497-25bdc000-1843-11eb-8f73-c937b3f479db.png)

I follow the convention to include related link in step guide.

> #### Alias Patterns for MQL queries

Reviewer 3: ```suggestion
#### Alias patterns for MQL queries
```

> MQL queries use the same [alias patterns as metric queries]({{< relref "#metric-queries" >}}).

Reviewer 3: ```suggestion
MQL queries use the same alias patterns as [metric queries]({{< relref "#metric-queries" >}}).
```
